bin/mpiexecjl

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env julia
+# -*- julia -*-
+#
+# Copyright (C) 2020 Mosè Giordano
+# License is MIT "Expat"
+#
+### Commentary:
+#
+# Command line utility to call the `mpiexec` binary used by the `MPI.jl` version
+# in the given Julia project.  It has the same syntax as the `mpiexec` binary
+# that would be called, with the additional `--project=...` to select a
+# different Julia project.
+#
+# Examples of usage (the MPI flags available depend on the MPI implementation
+# called):
+#
+#   $ mpiexecjl --version
+#   $ mpiexecjl -n 40 mpi-script.jl
+#   $ mpiexecjl --project=my_experiment -n 80 --oversubscribe mpi-script.jl
+#
+### Code:
+
+# From BinaryBuilder.jl
+function extract_flag(ARGS, flag, val = "")
+    flag_args = String[]
+    other_args = String[]
+    for f in ARGS
+        if startswith(flag, f)
+            push!(flag_args, f)
+        else
+            push!(other_args, f)
+        end
+    end
```

Comment:
```suggestion
    flag_args = filter(f->startswith(flag, f), ARGS)
    other_args = filter(f->!startswith(flag, f), ARGS)
```
Is it better to use `filter` here?

Reply:
I guess it requires two passes over the ARGS, but that's probably not a big deal. Would be nice if there was an easy way to do this.

Reply:
True